README.rst

```diff
@@ -9,6 +9,16 @@ Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/about>`_.
 Currently, Windows PowerShell, Command Prompt, and Git Shell for
 Windows are supported with limitations (See `Windows Issues`_).
 
+This product is supported by the Cybersecurity Development team at the 
+University of Illinois, on a best-effort basis. As of the last update to 
+this README, the expected End-of-Life and End-of-Support dates of this 
+product are 05 October 2025.
```

Comment:
Should we say product or version? Product seems much more final implying that this is the last version beyond minor bug fixes. Version is more open ended implying to me that there likely will be another version, but this current version will no longer be supported beyond October 5, 2025.

Reply:
I like that change. The implied finality has been bothering me - I think changing 'product' to 'version' is a simple way to more clearly communicate this.

Reply:
I used version in place of the second product, and I think it reads more as intended.